spec/register_spec.rb

```diff
@@ -187,7 +187,7 @@ def with_args(*args)
       begin
         $thor_runner = false
         help_output = capture(:stdout) { BoringVendorProvidedCLI.start(%w[exciting]) }
-        expect(help_output).to include("thor exciting_plugin_c_l_i fireworks")
+        expect(help_output).to include("thor exciting fireworks")
```

Comment:
Because ExcitingPluginCLI is registered with subcommand_name "exciting" by the following code:

```
BoringVendorProvidedCLI.register(
  ExcitingPluginCLI,
  "exciting",
  "do exciting things",
  "Various non-boring actions")
```

https://github.com/groovenauts/thor/blob/d31a6ef82cb155ef11f6326ac555070fd727df5e/spec/register_spec.rb#L99-L103